src/doc/gui_manual/Operators/OperatorTypes/Tessellate_operator.rst

```diff
@@ -0,0 +1,56 @@
+.. _Tessellate operator:
+
+Tessellate operator
+~~~~~~~~~~~~~~~~~~~
+
+The Tessellate operator is an operator that tessellates high order elements
+so that they appear curved.
+
+.. _tessellate:
+
+.. figure:: images/tessellate.png
+
+   Tessellate operator example
+
+The Tessellate operator supports the following high order element types.
+
+* QUADRATIC_EDGE
+* CUBIC_LINE
+* LAGRANGE_TRIANGLE
+* QUADRATIC_TRIANGLE
+* BIQUADRATIC_TRIANGLE
+* LAGRANGE_QUADRILATERAL
+* BIQUADRATIC_QUAD
+* QUADRATIC_QUAD
+* LAGRANGE_TETRAHEDRON
+* QUATRADIC_TETRA
+* LAGRANGE_HEXAHEDRON
+* QUADRATIC_HEXAHEDRON
+
+If the Tessellate operator encounters an unsupported element type it will
+remove the element from the mesh.
+
+Changing the tessellation accuracy
+""""""""""""""""""""""""""""""""""
+
+The tessalation accuracy is controlled by the **Chord error**. It is the
+ratio of a chord to the distance from the curve and is independent of the
+scale of the object. The default **Chord error** is 0.035, which will
+typically still exhibit some discritization. A **Chord error** of 0.01
```

Comment:
discritization -> discretization here and elsewhere

Reply:
There should only be TessellateAttributes. I originally named them TessellateOperatorAttributes, but then realized that this was inconsistent with the rest of VisIt. I decided to fix the inconsistency since this affects the Python interface and having an inconsistency in the Python interface would be bad.

Reply:
I now see that I accidentally left in the old classes associated with TessellateOperatorAttributes. I will remove them.